src/protagonist/Engine/Ingest/IngestExecutor.cs

```diff
@@ -27,6 +32,14 @@ public class IngestExecutor
         var workers = workerBuilder.GetWorkers(asset);
         
         var context = new IngestionContext(asset);
+        long? preIngestionAssetSize = null;

```

Comment:
Declaring the var outside isn't necessary as it's only used if the `if` statement is `true`

```cs
if (!assetIngestorSizeCheck.CustomerHasNoStorageCheck(asset.Customer))
{
    var preIngestionAssetSize = await assetRepository.GetImageSize(asset.Id, cancellationToken);
    context.WithPreIngestionAssetSize(preIngestionAssetSize);
}
```